kogito-maven-plugin/src/main/java/org/kie/kogito/maven/plugin/AbstractKieMojo.java

```diff
@@ -67,7 +88,7 @@
     protected File generatedResources;
 
     @Parameter(property = "kogito.codegen.persistence", defaultValue = "true")
-    protected boolean persistence;
+    protected String persistence;
```

Comment:
```suggestion
    protected String generatePersistence;
```
To keep cohesion between these properties?